### DISCUSSION TITLE
Daily 11/05

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#255] [Daily Sprint]
- Qué hice ayer?
- Qué haré hoy
- Dificultades o Impedimentos

**Comment by BraianTroncoso** [comment]
Hoy voy a continuar con una clase de Servicios que estoy viendo en un curso y terminar la clase de programación sobre Java.
Si tengo algo de tiempo me pongo a ver algo de Django ya para seguir con el back.
También tengo clases de programación sobre Python en el ISPC.

**Comment by NataliaAlvarezIspc** [comment]
Ayer no pude hacer mucho, ya que me tuve que ocupar de otros temas.
Hoy estaré realizando algunas tareas que me asigné y continuaré con los contenidos de la plataforma que aún no terminé. Por la tarde tengo clases de programación del Ispc.
